synapse/models/inet.py

```diff
@@ -695,487 +695,791 @@ def _revModl201710111553(self):
+                        'doc': 'The IMEI of the interface.'}),
+                    ('mob:imsi', {'ptype': 'tel:mob:imsi',
+                        'doc': 'The IMSI of the interface.'}),
                 )),
 
                 ('inet:wifi:ssid', {}, []),
 
                 ('inet:web:acct', {'ptype': 'inet:web:acct'}, [
-                    ('site', {'ptype': 'inet:fqdn', 'ro': 1}),
-                    ('user', {'ptype': 'inet:user', 'ro': 1}),
-
-                    ('dob', {'ptype': 'time'}),
-
-                    # ('bio:bt',{'ptype':'wtf','doc':'The web account's self documented blood type'}),
-
-                    ('url', {'ptype': 'inet:url'}),
-                    ('webpage', {'ptype': 'inet:url'}),
-                    ('avatar', {'ptype': 'file:bytes'}),
-
-                    ('tagline', {'ptype': 'str:txt', 'doc': 'A web account status/tag line text'}),
-                    ('loc', {'ptype': 'str:lwr', 'doc': 'The web account self declared location'}),
-                    ('occupation', {'ptype': 'str:lwr', 'doc': 'A web account self declared occupation'}),
+                    ('site', {'ptype': 'inet:fqdn', 'ro': 1,
+                        'doc': 'The site or service associated with the account.'}),
+                    ('user', {'ptype': 'inet:user', 'ro': 1,
+                        'doc': 'The unique identifier for the account (may be different from the common '
+                            'name or display name).'}),
+                    ('url', {'ptype': 'inet:url',
+                        'doc': 'The service provider URL where the account is hosted.'}),
+                    ('name', {'ptype': 'inet:user',
+                        'doc': 'The name associated with the account (may be different from the account '
+                            'identifier, e.g., a display name).'}),
+                    ('avatar', {'ptype': 'file:bytes',
+                        'doc': 'The file representing the avatar (e.g., profile picture) for the account.'}),
+                    ('tagline', {'ptype': 'str:txt',
+                        'doc': 'The text of the account status or tag line.'}),
+                    ('webpage', {'ptype': 'inet:url',
+                        'doc': 'A related URL specified by the account (e.g., a personal or company web '
+                             'page, blog, etc.).'}),
+                    ('loc', {'ptype': 'str:lwr',
+                        'doc': 'A self-declared location for the account.'}),
+                    ('occupation', {'ptype': 'str:lwr',
+                        'doc': 'A self-declared occupation for the account.'}),
+                    ('dob', {'ptype': 'time',
+                        'doc': 'A self-declared date of birth for the account (if the account belongs to a person).'}),
                     # ('gender',{'ptype':'inet:fqdn','ro':1}),
+                    # ('bio:bt',{'ptype':'wtf','doc':'The web account's self documented blood type'}),
```

Comment:
Should we just remove these two commented out lines?

Reply:
nah, lets leave these reminders for now.